Code/Framework/AzCore/AzCore/Memory/SystemAllocator.cpp

```diff
@@ -24,6 +24,8 @@
 
 #include <AzCore/Memory/HphaAllocator.h>
 
+#define USE_OS_ALLOCATE 1
```

Comment:
I'm wondering if we should wrap this in #if CARBONATED or use an AZ Trait

Reply:
yeah this should not be enabled by default. This needs perf test to ensure it didn't increase FE->BE time.